underfs/hdfs/pom.xml

```diff
@@ -85,13 +85,10 @@
       </build>
     </profile>
 
-    <!-- Profile to build this UFS module connecting to hdfs 2, this is the default profile -->
+    <!-- Profile to build this UFS module connecting to hdfs 2 -->
```

Comment:
Can we add enforcement for Java 8 if it is required by hdfs 3.3 and under?

Reply:
i don't think this is necessary, because for hdfs under 3.3, we can still use Java 11 to build to target 8.